frontend/src/pages/transactions/details/Details.tsx

```diff
@@ -24,20 +30,192 @@ import {
   useGetTransaction,
 } from "../../../hooks/use-api";
 import { FlowUtils } from "../../../utils/flow-utils";
+import { createColumnHelper } from "@tanstack/table-core";
+import {
+  SignableObject,
+  Transaction,
+} from "types/generated/entities/transactions";
+import { info } from "console";
+import Table from "../../../components/table/Table";
+import { DecoratedPollingEntity } from "frontend/src/hooks/use-timeout-polling";
+import { Event } from "types/generated/entities/events";
+import { ComputedEventData, EventUtils } from "../../../utils/event-utils";
+import CopyButton from "../../../components/copy-button/CopyButton";
+import CollapsibleCard from "../../../components/collapsible-card/CollapsibleCard";
+import { flexRender } from "@tanstack/react-table";
 
 type RouteParams = {
   transactionId: string;
 };
 
+// EVENTS SUBTABLE
+const columnHelperEvents = createColumnHelper<ComputedEventData>();
+const columnsEvents = [
+  columnHelperEvents.display({
+    id: "tableTitle",
+    header: () => <Label variant="medium">VALUES</Label>,
+  }),
+  columnHelperEvents.accessor("name", {
+    header: () => <Label variant="medium">NAME</Label>,
+    cell: (info) => (
+      <Value>
+        <Ellipsis className={classes.ellipsis}>{info.getValue()}</Ellipsis>
+      </Value>
+    ),
+  }),
+  columnHelperEvents.accessor("type", {
+    header: () => <Label variant="medium">TYPE</Label>,
+    cell: (info) => (
+      <Value>
+        <Ellipsis className={classes.ellipsis}>{info.getValue()}</Ellipsis>
+      </Value>
+    ),
+  }),
+  columnHelperEvents.accessor("value", {
+    header: () => <Label variant="medium">VALUE</Label>,
+    cell: (info) => (
+      <div>
+        <Ellipsis style={{ whiteSpace: "nowrap" }} className={classes.subtable}>
+          {info.getValue()}
+        </Ellipsis>
+        <CopyButton value={info.getValue()} />
+      </div>
+    ),
+  }),
+];
+
+// ENVELOPE SIGNATURES TABLE
+const columnsHelperEnvelope = createColumnHelper<SignableObject>();
+
+const columnsEnvelope = [
```

Comment:
Nit: Just a general note here. Comments are not always a good thing. Sometimes it's better to just "encode" the meaning of some code in the code itself - by providing a meaningfull variable name for example. Doing this can often make comments unecessary.

This post explains that pretty well: https://bpoplauschi.github.io/2021/01/20/Clean-Code-Comments-by-Uncle-Bob-part-2.html